### ISSUE TITLE
Feat: Ollama support

### ISSUE THREAD

**Issue by grigio** [wwardaww/gnome-gemini-ai#3]
is possible to add support to other LLMs like Ollama API ?

**Comment by wwardaww** [comment]
hi, I'm currently fulfilled with my freelance project but once I have some time, I will add other APIs (but not working in local LLMS)

**Comment by aayushrautela** [comment]
Yeah ollama api would be best, and will also go with foss spirit of linux.
